features/upgrade/api_auth/upgrade.feature

```diff
@@ -0,0 +1,51 @@
+Feature: apiserver and auth related upgrade check
+  # @author pmali@redhat.com
+  # @case_id OCP-22734
+  @upgrade-prepare
+  Scenario: Check Authentication operators and operands are upgraded correctly - prepare
+    # According to our upgrade workflow, we need an upgrade-prepare and upgrade-check for each scenario.
+    # But some of them do not need any prepare steps, which lead to errors "can not find scenarios" in the log.
+    # So we just add a simple/useless step here to get rid of the errors in the log.
+    Given the expression should be true> "True" == "True"
+
+  # @author pmali@redhat.com
+  # @case_id OCP-22734
+  @upgrade-check
+  @admin
+  Scenario: Check Authentication operators and operands are upgraded correctly
+    Given the "authentication" operator version matches the current cluster version
+
+    # Check cluster operators should be in correct status
+    Given the expression should be true> cluster_operator('authentication').condition(type: 'Progressing')['status'] == "False"
+    And the expression should be true> cluster_operator('authentication').condition(type: 'Available')['status'] == "True"
+    And the expression should be true> cluster_operator('authentication').condition(type: 'Degraded')['status'] == "False"
+    And the expression should be true> cluster_operator('authentication').condition(type: 'Upgradeable')['status'] == "True"
+    
+    # operator pod image
+    When I run the :get admin command with:
+      | resource  | po                                            |
+      | n         | openshift-authentication-operator             |
+      | o         | jsonpath={.items[0].spec.containers[0].image} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :auth_operator_image clipboard
+    
+    # Check cluster version
+    When I run the :get admin command with:
+      | resource | clusterversion/version           |
+      | o        | jsonpath={.status.desired.image} |
+    Then the step should succeed
+    And evaluation of `@result[:response]` is stored in the :payload_image clipboard
+
+    # Check the payload info
+    Given evaluation of `"oc adm release info --registry-config=/var/lib/kubelet/config.json <%= cb.payload_image %>"` is stored in the :oc_adm_release_info clipboard
+    When I store the ready and schedulable masters in the clipboard
+    And I use the "<%= cb.nodes[0].name %>" node
+    
+    # due to sensitive, didn't choose to dump and save the config.json file
+    # for using the step `I run the :oadm_release_info admin command ...`
+    
+    And I run commands on the host:
+      | <%= cb.oc_adm_release_info %> --image-for=cluster-authentication-operator      |
```

Comment:
Extra white spaces between `operator` and `|`

Reply:
@rhpmali , yes, core auto members have stricter requirement in format, therefore in future PRs pay more attention to space, indentation, alignment issues like this and others https://github.com/openshift/cucushift/pull/7695#discussion_r423415046 / https://github.com/openshift/cucushift/pull/7695#discussion_r423415134 to make the merge quicker.